client/my-sites/upgrades/domain-management/components/domain/maintenance-card.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React from 'react' ;
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import { getSelectedDomain, getTld } from 'lib/domains';
+import EmptyContent from 'components/empty-content';
+
+const MaintenanceCard = ( { domains, selectedDomainName, translate } ) => {
+	const domain = getSelectedDomain( { domains, selectedDomainName } ),
+		tld = getTld( domain.name );
```

Comment:
Ummm, you can simply use `selectedDomainName` :) That was the whole point of this - so that you don't need to go through the whole `getSelectedDomain` shuffle :)